framework/base/Model.php

```diff
@@ -876,6 +876,93 @@ public static function loadMultiple($models, $data, $formName = null)
     }
 
     /**
+     * Populates a set of models with the data from end user.
+     * This method is mainly used to collect tabular data input.
+     * The data to be loaded for each model is `$data[formName][index]`, where `formName`
+     * refers to the sort name of model class, and `index` the index of the model in the `$data` array.
+     * If `$formName` is empty, `$data[index]` will be used to populate each model.
+     * The data being populated to each model is subject to the safety check by [[setAttributes()]].
+     * @param array $data the data array. This is usually `$_POST` or `$_GET`, but can also be any valid array
+     * supplied by end user.
+     * @param string $formName the form name to be used for loading the data into the models.
+     * If not set, it will use the sort name of called class.
+     * @param static[] $origin original models to be populated. It will be check using `$keys` with supplied data.
+     * If same then will be used for result model.
+     * @param string|array $keys list of attribute to check is two model are equals. If `$keys` is `null`
+     * then it will use array index to check. If `$keys` is empty then always create new model.
+     * @param array $options options applied to model.
+     *
+     * - scenario: string, scenario applied to new model.
+     * - arguments: array, the parameters to be passed to the class constructor.
+     * 
+     * @return boolean|static[] whether at least one of the models is successfully populated.
+     */
+    public static function createMultiple($data, $formName = null, &$origin = [], $keys = null, $options = [])
+    {
+        $reflector = new \ReflectionClass(get_called_class());
+        $args = isset($options['arguments']) ? $options['arguments'] : [];
+        if ($formName === null) {
+            /* @var $model static */
+            $model = empty($args) ? new static() : $reflector->newInstanceArgs($args);
+            $formName = $model->formName();
+        }
+        if ($formName != '') {
+            $data = isset($data[$formName]) ? $data[$formName] : null;
+        }
+        if ($data === null) {
```

Comment:
I think createMultiple instead of false should return empty array or array with one new model, depending it is request or first page load. What about slight change like this? 
```
$oldData = $data;
if ($data === null) {
    if ($oldData === []) {
        return [$model]; 
	} else {
		return [];
	}
}
```
This will allow simplification controller:
```
public function actionCreate()
{
    $count = count(Yii::$app->request->post('Setting', []));
    $settings = [new Setting()];
    for($i = 1; $i < $count; $i++) {
        $settings[] = new Setting();
    }

    // ...
}
```
to this one:
```
public function actionCreate()
{
    $settings = Setting::createMultiple(Yii::$app->request->post());

    // ...
}
```